web-stories.php

```diff
@@ -42,7 +42,7 @@
 
 define( 'WEBSTORIES_VERSION', '1.3.0' );
 define( 'WEBSTORIES_DB_VERSION', '3.0.6' );
-define( 'WEBSTORIES_AMP_VERSION', '2.0.8' ); // Version of the AMP library included in the plugin.
+define( 'WEBSTORIES_AMP_VERSION', '2.0.10' ); // Version of the AMP library included in the plugin.
```

Comment:
`composer.json` says `2.0.8`. Should we perhaps change the requirement there from `^2.0.8` to a fixed `2.0.10`?